[2024-09-01T23:04:03.595+0000] {processor.py:161} INFO - Started process (PID=56) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:04:03.621+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:04:03.623+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:04:03.623+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:04:03.808+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:04:03.790+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:04:03.812+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:04:03.906+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.329 seconds
[2024-09-01T23:04:34.006+0000] {processor.py:161} INFO - Started process (PID=62) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:04:34.007+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:04:34.009+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:04:34.008+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:04:34.028+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:04:34.027+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:04:34.029+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:04:34.058+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.058 seconds
[2024-09-01T23:05:04.164+0000] {processor.py:161} INFO - Started process (PID=70) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:05:04.164+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:05:04.166+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:05:04.165+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:05:04.172+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:05:04.172+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:05:04.173+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:05:04.191+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.033 seconds
[2024-09-01T23:05:34.273+0000] {processor.py:161} INFO - Started process (PID=76) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:05:34.274+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:05:34.275+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:05:34.275+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:05:34.282+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:05:34.281+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:05:34.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:05:34.302+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.034 seconds
[2024-09-01T23:06:04.409+0000] {processor.py:161} INFO - Started process (PID=84) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:06:04.410+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:06:04.412+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:06:04.412+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:06:04.419+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:06:04.418+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:06:04.419+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:06:04.439+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.036 seconds
[2024-09-01T23:06:34.521+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:06:34.521+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:06:34.523+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:06:34.522+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:06:34.530+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:06:34.529+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:06:34.530+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:06:34.549+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.034 seconds
[2024-09-01T23:08:50.949+0000] {processor.py:161} INFO - Started process (PID=58) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:08:50.950+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:08:50.953+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:08:50.953+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:08:50.975+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:08:50.974+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from utils import setup_logging, load_credentials
ModuleNotFoundError: No module named 'utils'
[2024-09-01T23:08:50.976+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:08:51.010+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.075 seconds
[2024-09-01T23:09:21.128+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:21.128+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:09:21.130+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:21.130+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:21.140+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:21.139+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from utils import setup_logging, load_credentials
ModuleNotFoundError: No module named 'utils'
[2024-09-01T23:09:21.141+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:21.167+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.046 seconds
[2024-09-01T23:09:41.200+0000] {processor.py:161} INFO - Started process (PID=66) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:41.201+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:09:41.203+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:41.203+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:41.215+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:41.212+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from airflow.utils import setup_logging, load_credentials
ImportError: cannot import name 'setup_logging' from 'airflow.utils' (/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/__init__.py)
[2024-09-01T23:09:41.217+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:41.242+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.049 seconds
[2024-09-01T23:09:43.238+0000] {processor.py:161} INFO - Started process (PID=67) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:43.239+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:09:43.240+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:43.240+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:43.249+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:43.247+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from airflow.utils import setup_logging, load_credentials
ImportError: cannot import name 'setup_logging' from 'airflow.utils' (/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/__init__.py)
[2024-09-01T23:09:43.251+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:43.272+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.039 seconds
[2024-09-01T23:09:44.279+0000] {processor.py:161} INFO - Started process (PID=68) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:44.279+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:09:44.281+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:44.280+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:44.291+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:09:44.288+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from airflow.utils import setup_logging, load_credentials
ImportError: cannot import name 'setup_logging' from 'airflow.utils' (/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/__init__.py)
[2024-09-01T23:09:44.293+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:09:44.313+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.040 seconds
[2024-09-01T23:10:09.392+0000] {processor.py:161} INFO - Started process (PID=75) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:09.392+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:10:09.394+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:09.393+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:09.402+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:09.400+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from airflow.utils import setup_logging, load_credentials
ImportError: cannot import name 'setup_logging' from 'airflow.utils' (/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/__init__.py)
[2024-09-01T23:10:09.404+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:09.423+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.036 seconds
[2024-09-01T23:10:12.410+0000] {processor.py:161} INFO - Started process (PID=76) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:12.411+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:10:12.412+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:12.412+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:12.422+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:12.419+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from airflow.utils import setup_logging, load_credentials
ImportError: cannot import name 'setup_logging' from 'airflow.utils' (/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/__init__.py)
[2024-09-01T23:10:12.426+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:12.451+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.045 seconds
[2024-09-01T23:10:15.460+0000] {processor.py:161} INFO - Started process (PID=77) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:15.461+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:10:15.462+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:15.462+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:15.471+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:15.470+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 9, in <module>
    from utils import setup_logging, load_credentials
ModuleNotFoundError: No module named 'utils'
[2024-09-01T23:10:15.472+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:15.492+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.036 seconds
[2024-09-01T23:10:44.590+0000] {processor.py:161} INFO - Started process (PID=85) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:44.591+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:10:44.592+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:44.592+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:44.600+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:44.599+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 10, in <module>
    from spotify import authenticate_spotify, get_tracks_data, create_dataframe
ModuleNotFoundError: No module named 'spotify'
[2024-09-01T23:10:44.600+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:44.620+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.034 seconds
[2024-09-01T23:10:46.605+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:46.606+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:10:46.608+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:46.607+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:46.639+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:10:46.992+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:46.992+0000] {override.py:1829} INFO - Created Permission View: can edit on DAG:spotify_to_redshift_dag
[2024-09-01T23:10:47.007+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:47.007+0000] {override.py:1829} INFO - Created Permission View: can read on DAG:spotify_to_redshift_dag
[2024-09-01T23:10:47.022+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:47.022+0000] {override.py:1829} INFO - Created Permission View: can delete on DAG:spotify_to_redshift_dag
[2024-09-01T23:10:47.023+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:47.023+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:10:47.032+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:47.031+0000] {dag.py:3118} INFO - Creating ORM DAG for spotify_to_redshift_dag
[2024-09-01T23:10:47.032+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:10:47.032+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:10:47.055+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.454 seconds
[2024-09-01T23:11:17.162+0000] {processor.py:161} INFO - Started process (PID=100) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:11:17.163+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:11:17.164+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:11:17.164+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:11:17.179+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:11:17.202+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:11:17.202+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:11:17.212+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:11:17.212+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:11:17.231+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.073 seconds
[2024-09-01T23:11:47.277+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:11:47.277+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:11:47.279+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:11:47.278+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:11:47.290+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:11:47.313+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:11:47.313+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:11:47.324+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:11:47.324+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:11:47.346+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.073 seconds
[2024-09-01T23:12:17.443+0000] {processor.py:161} INFO - Started process (PID=114) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:12:17.444+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:12:17.446+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:12:17.445+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:12:17.463+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:12:17.491+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:12:17.491+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:12:17.501+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:12:17.501+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:12:17.520+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.083 seconds
[2024-09-01T23:12:47.547+0000] {processor.py:161} INFO - Started process (PID=121) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:12:47.547+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:12:47.549+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:12:47.548+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:12:47.560+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:12:47.582+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:12:47.582+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:12:47.592+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:12:47.592+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:12:47.612+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.069 seconds
[2024-09-01T23:13:17.767+0000] {processor.py:161} INFO - Started process (PID=127) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:13:17.768+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:13:17.769+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:13:17.769+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:13:17.839+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:13:17.888+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:13:17.888+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:13:17.903+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:13:17.903+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:13:17.949+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.213 seconds
[2024-09-01T23:13:48.072+0000] {processor.py:161} INFO - Started process (PID=134) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:13:48.073+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:13:48.074+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:13:48.074+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:13:48.091+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:13:48.113+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:13:48.113+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:13:48.123+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:13:48.123+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:13:48.142+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.074 seconds
[2024-09-01T23:14:18.200+0000] {processor.py:161} INFO - Started process (PID=142) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:14:18.201+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:14:18.202+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:14:18.202+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:14:18.213+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:14:18.236+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:14:18.236+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:14:18.246+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:14:18.246+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:14:18.264+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.069 seconds
[2024-09-01T23:14:48.341+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:14:48.342+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:14:48.343+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:14:48.343+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:14:48.355+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:14:48.377+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:14:48.376+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:14:48.387+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:14:48.387+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:14:48.405+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.070 seconds
[2024-09-01T23:15:18.465+0000] {processor.py:161} INFO - Started process (PID=156) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:15:18.469+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:15:18.471+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:15:18.470+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:15:18.490+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:15:18.633+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:15:18.633+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:15:18.642+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:15:18.641+0000] {dag.py:3118} INFO - Creating ORM DAG for spotify_to_redshift_dag
[2024-09-01T23:15:18.642+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:15:18.642+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:15:18.659+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.202 seconds
[2024-09-01T23:16:28.018+0000] {processor.py:161} INFO - Started process (PID=56) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:16:28.018+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:16:28.037+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:16:28.037+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:16:28.074+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:16:28.283+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:16:28.283+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:16:28.303+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:16:28.303+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:16:28.330+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.336 seconds
[2024-09-01T23:16:58.463+0000] {processor.py:161} INFO - Started process (PID=62) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:16:58.464+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:16:58.466+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:16:58.465+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:16:58.483+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:16:58.511+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:16:58.511+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:16:58.521+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:16:58.521+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:16:58.540+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.083 seconds
[2024-09-01T23:17:28.579+0000] {processor.py:161} INFO - Started process (PID=69) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:28.589+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:17:28.591+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:28.590+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:28.619+0000] {processor.py:840} INFO - DAG(s) 'spotify_to_redshift_dag' retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:28.652+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:28.652+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2024-09-01T23:17:28.669+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:28.668+0000] {dag.py:3954} INFO - Setting next_dagrun for spotify_to_redshift_dag to None, run_after=None
[2024-09-01T23:17:28.701+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.128 seconds
[2024-09-01T23:17:29.753+0000] {processor.py:161} INFO - Started process (PID=70) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:29.754+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:17:29.761+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:29.761+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:29.783+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:29.781+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:17:29.793+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:29.833+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.095 seconds
[2024-09-01T23:17:59.941+0000] {processor.py:161} INFO - Started process (PID=77) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:59.941+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:17:59.943+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:59.942+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:59.950+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:17:59.949+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:17:59.950+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:17:59.969+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.033 seconds
[2024-09-01T23:18:30.053+0000] {processor.py:161} INFO - Started process (PID=84) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:18:30.053+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:18:30.055+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:18:30.054+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:18:30.061+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:18:30.061+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:18:30.062+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:18:30.082+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.034 seconds
[2024-09-01T23:19:00.160+0000] {processor.py:161} INFO - Started process (PID=92) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:19:00.161+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:19:00.162+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:19:00.162+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:19:00.169+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:19:00.168+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:19:00.169+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:19:00.189+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.033 seconds
[2024-09-01T23:19:30.283+0000] {processor.py:161} INFO - Started process (PID=99) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:19:30.283+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:19:30.284+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:19:30.284+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:19:30.290+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:19:30.290+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:19:30.291+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:19:30.309+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.031 seconds
[2024-09-01T23:20:00.395+0000] {processor.py:161} INFO - Started process (PID=107) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:20:00.396+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:20:00.397+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:20:00.396+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:20:00.403+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:20:00.402+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:20:00.404+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:20:00.422+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.031 seconds
[2024-09-01T23:20:30.519+0000] {processor.py:161} INFO - Started process (PID=114) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:20:30.520+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:20:30.521+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:20:30.520+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:20:30.527+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:20:30.526+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:20:30.528+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:20:30.546+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.031 seconds
[2024-09-01T23:21:00.639+0000] {processor.py:161} INFO - Started process (PID=120) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:21:00.639+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:21:00.640+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:21:00.640+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:21:00.647+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:21:00.646+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:21:00.647+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:21:00.671+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.037 seconds
[2024-09-01T23:21:30.803+0000] {processor.py:161} INFO - Started process (PID=127) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:21:30.803+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:21:30.804+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:21:30.804+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:21:30.811+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:21:30.810+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:21:30.812+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:21:30.830+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.032 seconds
[2024-09-01T23:22:00.932+0000] {processor.py:161} INFO - Started process (PID=134) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:22:00.933+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:22:00.933+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:22:00.933+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:22:00.955+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:22:00.954+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:22:00.956+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:22:01.003+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.085 seconds
[2024-09-01T23:22:31.039+0000] {processor.py:161} INFO - Started process (PID=141) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:22:31.040+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:22:31.040+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:22:31.040+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:22:31.048+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:22:31.046+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:22:31.048+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:22:31.067+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.032 seconds
[2024-09-01T23:23:01.163+0000] {processor.py:161} INFO - Started process (PID=149) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:23:01.163+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:23:01.164+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:23:01.164+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:23:01.171+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:23:01.170+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:23:01.171+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:23:01.190+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.033 seconds
[2024-09-01T23:23:31.284+0000] {processor.py:161} INFO - Started process (PID=157) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:23:31.285+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:23:31.286+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:23:31.286+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:23:31.293+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:23:31.292+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:23:31.293+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:23:31.313+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.034 seconds
[2024-09-01T23:26:56.895+0000] {processor.py:161} INFO - Started process (PID=58) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:26:56.903+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:26:56.905+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:26:56.905+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:26:56.952+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:26:56.951+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:26:56.957+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:26:57.007+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.123 seconds
[2024-09-01T23:27:27.176+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:27:27.177+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:27:27.179+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:27:27.179+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:27:27.199+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:27:27.197+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:27:27.200+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:27:27.248+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.088 seconds
[2024-09-01T23:28:33.683+0000] {processor.py:161} INFO - Started process (PID=60) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:28:33.689+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:28:33.691+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:28:33.690+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:28:33.712+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:28:33.710+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:28:33.712+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:28:33.766+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.089 seconds
[2024-09-01T23:29:03.899+0000] {processor.py:161} INFO - Started process (PID=67) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:29:03.900+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:29:03.902+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:29:03.901+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:29:03.919+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:29:03.918+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:29:03.920+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:29:03.958+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.073 seconds
[2024-09-01T23:29:34.043+0000] {processor.py:161} INFO - Started process (PID=75) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:29:34.053+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:29:34.055+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:29:34.054+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:29:34.069+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:29:34.068+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:29:34.069+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:29:34.113+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.084 seconds
[2024-09-01T23:31:48.275+0000] {processor.py:161} INFO - Started process (PID=59) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:31:48.281+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:31:48.293+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:31:48.293+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:31:48.376+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:31:48.374+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:31:48.388+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:31:48.446+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.219 seconds
[2024-09-01T23:32:18.547+0000] {processor.py:161} INFO - Started process (PID=65) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:32:18.547+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:32:18.549+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:32:18.549+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:32:18.559+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:32:18.557+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:32:18.559+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:32:18.585+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.044 seconds
[2024-09-01T23:32:48.729+0000] {processor.py:161} INFO - Started process (PID=72) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:32:48.730+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:32:48.733+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:32:48.732+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:32:48.762+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:32:48.761+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:32:48.767+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:32:48.813+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.101 seconds
[2024-09-01T23:34:47.973+0000] {processor.py:161} INFO - Started process (PID=58) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:34:47.977+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:34:48.016+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:34:47.984+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:34:48.110+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:34:48.104+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:34:48.126+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:34:48.214+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.274 seconds
[2024-09-01T23:35:18.314+0000] {processor.py:161} INFO - Started process (PID=64) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:35:18.315+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:35:18.317+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:35:18.316+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:35:18.326+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:35:18.325+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:35:18.326+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:35:18.347+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.038 seconds
[2024-09-01T23:35:48.451+0000] {processor.py:161} INFO - Started process (PID=71) to work on /opt/airflow/dags/main_dags.py
[2024-09-01T23:35:48.452+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/main_dags.py for tasks to queue
[2024-09-01T23:35:48.453+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:35:48.453+0000] {dagbag.py:545} INFO - Filling up the DagBag from /opt/airflow/dags/main_dags.py
[2024-09-01T23:35:48.463+0000] {logging_mixin.py:188} INFO - [2024-09-01T23:35:48.462+0000] {dagbag.py:355} ERROR - Failed to import: /opt/airflow/dags/main_dags.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/dagbag.py", line 351, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 995, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/opt/airflow/dags/main_dags.py", line 8, in <module>
    from config import REDSHIFT_HOST, REDSHIFT_PORT, REDSHIFT_DBNAME, REDSHIFT_USER, YEARS
ModuleNotFoundError: No module named 'config'
[2024-09-01T23:35:48.463+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/main_dags.py
[2024-09-01T23:35:48.482+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/main_dags.py took 0.037 seconds
